doc/api/util.md

```diff
@@ -1393,6 +1393,30 @@ added: v10.0.0
 
 Returns `true` if the value is a native `External` value.
 
+```c++
+#include <js_native_api.h>
+#include <string.h>
+napi_value result;
+static napi_value MyNapi(napi_env env, napi_callback_info info) {
+  int *raw = (int*) malloc(1024);
+  NAPI_CALL(env, napi_create_external(env, (void *) raw, NULL, NULL, &result));
+  return result;
+}
+...
+DECLARE_NAPI_PROPERTY("myNapi", MyNapi)
+...
+```
+...
+```js
+const native = require('napi_addon.node');
+const data = native.myNapi();
+util.types.isExternal(data); // returns true
+utils.types.isExternal(0); // returns false
+util.types.isExternal(new Number(10)); // returns false
```

Comment:
```suggestion
```

I would just remove this. The other examples should hopefully already be enough.

Reply:
ditto